src/main/java/com/robinpowered/sdk/model/Invitee.java

```diff
@@ -27,12 +27,20 @@
     public enum ResponseStatus {
 
         /**
+         * [Deprecated]
```

Comment:
Move this below the description and use javadoc. 

`@deprecated Use `UNKNOWN` instead.`